src/icon/mixins.scss

```diff
@@ -4,81 +4,72 @@
 
 @mixin iui-icons-default {
   @include iui-icons-color();
+  display: inline-flex;
```

Comment:
thats huge...
why `inline-flex` vs `flex` ?

Reply:
`flex` would make it a block-level element, which is different from how it previously behaves